asv_bench/benchmarks/stat_ops.py

```diff
@@ -1,205 +1,115 @@
-from .pandas_vb_common import *
+import numpy as np
+import pandas as pd
 
-
-def _set_use_bottleneck_False():
-    try:
-        pd.options.compute.use_bottleneck = False
-    except:
-        from pandas.core import nanops
-        nanops._USE_BOTTLENECK = False
+from .pandas_vb_common import setup  # noqa
 
 
 class FrameOps(object):
-    goal_time = 0.2
 
+    goal_time = 0.2
     param_names = ['op', 'use_bottleneck', 'dtype', 'axis']
-    params = [['mean', 'sum', 'median'],
+    params = [['mean', 'sum', 'median', 'std'],
```

Comment:
you could make use_bottleneck a param here

Reply:
use_bottleneck is a param here, over `[True, False]`